### PR TITLE
Kk cloudv3 deprecate

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**
@Sakshi-Dhakad @MarkvanMents I've gone through our docs and removed most references to cloud v3. I've kept the following:

https://docs.mendix.com/developerportal/deploy/securing-outgoing-connections-from-your-application/ (referenced in a historical context)

https://docs.mendix.com/developerportal/deploy/mxcloudv4/ (links to instructions on migrating to v4)

https://docs.mendix.com/developerportal/deploy/migrating-to-v4/ (instructions on migrating to v4)

https://docs.mendix.com/releasenotes/developer-portal/mendix-cloud/ (old release notes)

https://docs.mendix.com/releasenotes/studio-pro/8.0/ (even older release notes)

Let me know if anything needs correcting, or if can be merged. NB I had to remove a few articles completely, but I've kept a backup in case it's needed.

### CODE REVIEW

**Comment by Sakshi-Dhakad** [comment]
Thanks Katarzyna, makes sense, thanks for the update 😊

Cheers,

Sakshi Dhakad
Mendix | Product Manager | Mendix Cloud

From: katarzyna-koltun-mx ***@***.***>
Sent: Thursday, 15 December 2022 14:01
To: mendix/docs ***@***.***>
Cc: Sakshi Dhakad ***@***.***>; Mention ***@***.***>
Subject: [mendix/docs] Kk cloudv3 deprecate (PR #5426)


@Sakshi-Dhakad<https://github.com/Sakshi-Dhakad> @MarkvanMents<https://github.com/MarkvanMents> I've gone through our docs and removed most references to cloud v3. I've kept the following:

https://docs.mendix.com/developerportal/deploy/securing-outgoing-connections-from-your-application/ (referenced in a historical context)

https://docs.mendix.com/developerportal/deploy/mxcloudv4/ (links to instructions on migrating to v4)

https://docs.mendix.com/developerportal/deploy/migrating-to-v4/ (instructions on migrating to v4)

https://docs.mendix.com/releasenotes/developer-portal/mendix-cloud/ (old release notes)

https://docs.mendix.com/releasenotes/studio-pro/8.0/ (even older release notes)

Let me know if anything needs correcting, or if can be merged. NB I had to remove a few articles completely, but I've kept a backup in case it's needed.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/mendix/docs/pull/5426

Commit Summary

  *   dc81d45<https://github.com/mendix/docs/pull/5426/commits/dc81d457e715361bceec51356b75a8c96a8b215c> Removing information about cloud v3
  *   809aa3a<https://github.com/mendix/docs/pull/5426/commits/809aa3af9460ec822e908cb7b8f5166ead8ec525> cloud v3 deprecate
  *   ea3a8cb<https://github.com/mendix/docs/pull/5426/commits/ea3a8cb2217118fd8fa227271c158436cb65c530> cloud v3 deprecation
  *   e341007<https://github.com/mendix/docs/pull/5426/commits/e3410078ec1578a5d347b3bc6717cfa3bda19f23> Merge branch 'development' into kk-cloudv3-deprecate

File Changes

(33 files<https://github.com/mendix/docs/pull/5426/files>)

  *   M content/en/docs/addons/apd-addon/ig/ig-three.md<https://github.com/mendix/docs/pull/5426/files#diff-fbab5dbac4fcfa0d32c54200ce249ae8d534473911672baab487145b009bc3ab> (5)
  *   M content/en/docs/addons/apd-addon/ig/ig-two.md<https://github.com/mendix/docs/pull/5426/files#diff-3dff79f6256410e9dadaab1c0ba3523c5195ce839b4e165e0c01d7936538a63c> (5)
  *   M content/en/docs/developerportal/deploy/general/securing-outgoing-connections-from-your-application.md<https://github.com/mendix/docs/pull/5426/files#diff-456614c98920cc72b77606bfdb3421bea0b203953c571845669354667803f040> (2)
  *   M content/en/docs/developerportal/deploy/mendix-cloud-deploy/_index.md<https://github.com/mendix/docs/pull/5426/files#diff-0ed8586bd697338e9f326b26704c52c54cceee9126b0a64c7fc69b6f3df64004> (2)
  *   M content/en/docs/developerportal/deploy/mendix-cloud-deploy/access-restrictions.md<https://github.com/mendix/docs/pull/5426/files#diff-0574815c506ae293b41b8a4fa614638e90dd14b210059e9e1d051944a705737a> (42)
  *   M content/en/docs/developerportal/deploy/mendix-cloud-deploy/cloud-version-region.md<https://github.com/mendix/docs/pull/5426/files#diff-314daf2f4c82c0e116028d20f5ab7e40abb6085c810acee00bb145d049e7b3a3> (6)
  *   M content/en/docs/developerportal/deploy/mendix-cloud-deploy/environments-details.md<https://github.com/mendix/docs/pull/5426/files#diff-eda9d45be848a3f8dd2e100daa92d94bde0f0e680934905404b88c8c62953498> (39)
  *   M content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-cloud-status.md<https://github.com/mendix/docs/pull/5426/files#diff-92460722afed007a0595c0d3639bb2f468a768668a835a0f354eb102c34f136d> (2)
  *   M content/en/docs/developerportal/deploy/mendix-cloud-deploy/mendix-ip-addresses.md<https://github.com/mendix/docs/pull/5426/files#diff-e99f5c635793dcccc041a4d5f56742442b2ec369cebbcfe3627933afba9ec853> (11)
  *   M content/en/docs/developerportal/deploy/mendix-cloud-deploy/mxcloudv4/scale-environment.md<https://github.com/mendix/docs/pull/5426/files#diff-5e7b6f5b8b53885b45628727a7f8c8182fe2db9b2f265c4d43ddc49f046c7e74> (1)
  *   M content/en/docs/developerportal/deploy/mendix-cloud-deploy/sending-email.md<https://github.com/mendix/docs/pull/5426/files#diff-9027c10c398edc0b41c993a73d4f84f20577a17d6058ce47679fece251f65524> (61)
  *   M content/en/docs/developerportal/operate/backups/_index.md<https://github.com/mendix/docs/pull/5426/files#diff-56b1433b4486612a90363a40ceda7ac2d3caf4a20beb53cf039833c01f0f3d5c> (19)
  *   M content/en/docs/developerportal/operate/backups/restore-backup.md<https://github.com/mendix/docs/pull/5426/files#diff-105aa40194159250025c307f7338ea1c44143edf5261fb4869f0b95c845f5988> (7)
  *   M content/en/docs/developerportal/operate/logs.md<https://github.com/mendix/docs/pull/5426/files#diff-33983094b97093549b82c93e23ac2a35ebf1c9528089da25255c7ee73734792e> (2)
  *   M content/en/docs/developerportal/operate/metrics/_index.md<https://github.com/mendix/docs/pull/5426/files#diff-bded2eafb09b1abff7bee31fe6c8c2c87bb280209102afd289914dfa50786a72> (1)
  *   D content/en/docs/developerportal/operate/metrics/calculate-diskspace-of-a-cloud-app-environment.md<https://github.com/mendix/docs/pull/5426/files#diff-0ccfdb12636b495a096aa58e75f48057d71a20285bf5bdd55ba58cec084b6856> (83)
  *   M content/en/docs/developerportal/operate/metrics/monitoring-with-apm/_index.md<https://github.com/mendix/docs/pull/5426/files#diff-8941b8ec559b7067a878bef1654ee8466e0d0396a496b5baa3c5c1bf5fd44d42> (4)
  *   M content/en/docs/developerportal/operate/metrics/monitoring-with-apm/appdynamics-metrics.md<https://github.com/mendix/docs/pull/5426/files#diff-3737e6613119ce8886f50f356d59b4a791df77d7378601b865472190e8d07c9f> (4)
  *   M content/en/docs/developerportal/operate/metrics/monitoring-with-apm/datadog-metrics.md<https://github.com/mendix/docs/pull/5426/files#diff-eed9cf01edd09b675aa07d9723edac8f81206a768102cd316911c6478c03f4ee> (4)
  *   M content/en/docs/developerportal/operate/metrics/monitoring-with-apm/splunk-metrics.md<https://github.com/mendix/docs/pull/5426/files#diff-ed9431640603ce430c90dfa38a8e9e88a11d89691f0078b7f6916dbbd6b33679> (4)
  *   M content/en/docs/developerportal/operate/metrics/trends-v4.md<https://github.com/mendix/docs/pull/5426/files#diff-e548a450980a06eebaef1483962f490cefe56c89e9b3eaca14494e14094b4d59> (3)
  *   D content/en/docs/developerportal/operate/metrics/trends.md<https://github.com/mendix/docs/pull/5426/files#diff-9159643281c3a79f97a0405de6ffa42a555a47e5353c3711808a19af6faf1b45> (321)
  *   M content/en/docs/developerportal/operate/metrics/troubleshooting-mxcloud-runningnow.md<https://github.com/mendix/docs/pull/5426/files#diff-3c71ab2d64a05849cd3df68e39986e993c0946f2bb6befa043ddc1e8a28ebc27> (1)
  *   M content/en/docs/developerportal/operate/monitoring-application-health/_index.md<https://github.com/mendix/docs/pull/5426/files#diff-4fea20daf419ee0ef8f91a76ad73763620bbfc85815ab5ee6432d3f43fbad1a4> (93)
  *   M content/en/docs/developerportal/operate/monitoring-application-health/receive-alerts.md<https://github.com/mendix/docs/pull/5426/files#diff-8a4da17b177d7c4c00aa522cf6b10264910c926160727730685d526b3540c569> (1)
  *   M content/en/docs/howto8/monitoring-troubleshooting/debug-microflows/debug-microflows-remotely.md<https://github.com/mendix/docs/pull/5426/files#diff-6ffd24b35eba013273109f52a8aa499117e6898f380a0b812f7212de5335c1be> (4)
  *   M content/en/docs/refguide/mobile/distributing-mobile-apps/logging.md<https://github.com/mendix/docs/pull/5426/files#diff-d089f4d4361aa2f9dd769f58da7ceec00af7d0c299d1c05b960b0c8cb634165c> (2)
  *   M content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/debug-microflows-and-nanoflows/debug-microflows-remotely.md<https://github.com/mendix/docs/pull/5426/files#diff-c274eed8840093785e72179941b7177f6e6a2b22f2ed347369a0f6c268029e79> (4)
  *   M content/en/docs/refguide/runtime/custom-settings/tricky-custom-runtime-settings.md<https://github.com/mendix/docs/pull/5426/files#diff-59e07d9e7e319c8a0dd03a1cc2e9cd6ad9a912a74bed5b501f58355fcaaa1431> (2)
  *   M content/en/docs/refguide7/runtime/custom-settings/tricky-custom-runtime-settings.md<https://github.com/mendix/docs/pull/5426/files#diff-f34027205c1844f0a80fd1efdbcc59a25ef8a21ce0d89997e2d790d5ade27437> (2)
  *   M content/en/docs/refguide8/general/moving-from-7-to-8/_index.md<https://github.com/mendix/docs/pull/5426/files#diff-05155c3401937491d7de2abe5a2c3195382d64d0fd951ade5e7f4ed7d0159f63> (4)
  *   M content/en/docs/refguide8/runtime/custom-settings/tricky-custom-runtime-settings.md<https://github.com/mendix/docs/pull/5426/files#diff-c50e8803cb4185240b19abe62ad7ee96d54412653f9384901596d0ae7fb52dac> (2)
  *   M content/en/docs/releasenotes/deployment/mendix-cloud.md<https://github.com/mendix/docs/pull/5426/files#diff-8f889226779c91ddc58c5c276d5feecb6770988c90e0185a66ae5345e0443bb9> (2)

Patch Links:

  *   https://github.com/mendix/docs/pull/5426.patch
  *   https://github.com/mendix/docs/pull/5426.diff

File: content/en/docs/developerportal/deploy/general/securing-outgoing-connections-from-your-application.md [hunk 1]
```diff
@@ -63,8 +63,6 @@ This scenario is identical to scenario 4, with one exception: your app includes
 
 {{% alert color="warning" %}}
 VPN is not supported at all in Mendix Cloud V4.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/general/securing-outgoing-connections-from-your-application.md]: Perhaps:
```suggestion
VPN is not supported at all in the Mendix Cloud.
```

File: content/en/docs/developerportal/operate/logs.md [hunk 2]
```diff
@@ -48,8 +48,6 @@ For apps deployed to the deprecated Mendix Cloud version 3 (v3), you can select
 
 You can view the current log in your browser, or you can view and download an archived log. All timestamps are shown in UTC.
  
-Logs on Mendix Cloud v3 are rotated and archived every 5 MB. The logs are kept for at least 3 months.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/operate/logs.md:51]: I wonder if we can remove the whole of section 3: **Apps Deployed to Mendix Cloud Version 3** as I don't think the logs will be relevant once the app has been migrated.

And the "v3" tag on this file?

File: content/en/docs/developerportal/deploy/general/securing-outgoing-connections-from-your-application.md [hunk 3]
```diff
@@ -63,8 +63,6 @@ This scenario is identical to scenario 4, with one exception: your app includes
...
-
-VPN is deprecated in Mendix Cloud V3. Although you can still set it up, you will not receive support from Mendix for a VPN in Mendix Cloud V3, or for using VPN in deployments to other clouds.
 {{% /alert %}}
 
 Many companies use a Virtual Private Network (VPN) to secure network connections between their own offices and data centers and other companies.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/general/securing-outgoing-connections-from-your-application.md:68]: I wonder if we can be more brutal about this and just remove the whole of the next paragraph as being irrelevant to current customers? I.e. the paragraph:
`Mendix offered IPSEC VPN in the Mendix Cloud V3 on some legacy connections, but is currently moving towards other secure protocols. These work better with modern Cloud-native best practices such as scaling horizontally, failover to other data centers, and multi-tenant cloud environments. It was removed in Mendix Cloud V4.`

**Review by MarkvanMents — COMMENTED** [review]
Great and thorough removal of the v3 documents @katarzyna-koltun-mx and a good choice of things to keep.
I have made a couple of minor suggestions but nothing to stop this being published.

The only thing that crosses my mind is whether we should still be using "Mendix Cloud v4" now that there is only one sort of Mendix Cloud. I think we probably should, as new developments in the future might mean we have two sorts of cloud again and will want to distinguish them if that were to happen.

We can always change this decision later, if we want. And we wouldn't want to do it immediately as we have still left a couple of documents about migration.

**Review by MarkvanMents — APPROVED** [review]

File: content/en/docs/developerportal/deploy/general/securing-outgoing-connections-from-your-application.md (hunk 3 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/deploy/general/securing-outgoing-connections-from-your-application.md:68]: Done.

File: content/en/docs/developerportal/deploy/general/securing-outgoing-connections-from-your-application.md (hunk 1 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/deploy/general/securing-outgoing-connections-from-your-application.md]: Done.

File: content/en/docs/developerportal/operate/logs.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/operate/logs.md:51]: Done.

**Comment by katarzyna-koltun-mx** [comment]
@MarkvanMents @Sakshi-Dhakad I've merged the PR. The changes should appear on production tomorrow.

**Comment by Sakshi-Dhakad** [comment]
Thanks @katarzyna-koltun-mx !

**Pull request merged**